requirements-dev.txt

```diff
@@ -0,0 +1,7 @@
+click==7.1.2
```

Comment:
I hate that these are separate files. Why isn't python a normal language.

Reply:
We could use a dedicated dependency management tool instead of `pip`
I have used `poetry` before but I am a little hesitant to bring it in. Just adds another layer of complexity (it also creates virtual env etc etc).
https://github.com/python-poetry/poetry

Reply:
lol